src/main/java/uk/gov/di/ipv/cri/common/library/service/PersonIdentityMapper.java

```diff
@@ -339,6 +339,7 @@ private List<CanonicalAddress> mapAddresses(List<Address> addresses) {
                             canonicalAddress.setDependentStreetName(a.getDependentStreetName());
                             canonicalAddress.setStreetName(a.getStreetName());
                             canonicalAddress.setAddressCountry(a.getAddressCountry());
+                            canonicalAddress.setAddressRegion(a.getAddressRegion());
```

Comment:
I assume this will work if region is null?

Reply:
Is there an existing test where the region is null? I assume yes if we have tests for UK addresses

Reply:
Yeah there are tests where `addressRegion` is `null`, but no specific `assertNull` test, I suppose I could add one.

Reply:
Yes, please